### DISCUSSION TITLE
Is it possible to get pdf from url with custom header?

### DISCUSSION THREAD

**Discussion by HananoshikaYomaru** [afreakyelf/Pdf-Viewer#11] [General]
I need to request a pdf from an api and I need to send the request with custom header. Is it possible to do this with this library?

**Comment by afreakyelf** [comment]
Hi,
   You can pass the Pdf url after getting the response from api.
But you can't request api.
Thanks for checking out the library, do let me know if you have any doubt.

**Comment by afreakyelf** [comment]
Hi, This has been resolved in new upcoming release. Thankk you for checking out the library.
